SPECS/azurelinux-image-tools/generate_source_tarball.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
```

Comment:
is that a helper script to run when uploading a new source tarball?

Reply:
Yes, but locally. Once the source tarball and vendor tarball get generated, these files should be uploaded to the blob.